profiles/api.py

```diff
@@ -96,6 +96,8 @@ class ProfileViewSet(viewsets.ModelViewSet):
     renderer_classes = [JSONRenderer, BrowsableAPIRenderer]
 
     def get_queryset(self):
+        if self.request.user.is_staff:
```

Comment:
`.is_staff` might mean some very restricted permissions too, it's not `.is_superuser`. I would not probably allow any staff member to see all profiles by default.

Reply:
Yeah, very good point! Anyway we'll probably have to add more fine grained permission classes for youth officers and such.

Reply:
it should probably be something along the lines of `if self.request.user.has_permissions()`.